src/styles/blocks/_menu.scss

```diff
@@ -0,0 +1,78 @@
+.menu {
+  background-color: $menu-bg;
+
+  &__content {
+    height: 100vh;
+    padding: $header-top-padding 0;
+
+    @include onTablet {
+      padding: $header-padding-desktop 0;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    list-style-type: none;
+    margin-top: 44px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 8px 0;
+    border-bottom: 1px solid transparent;
+
+    text-decoration: none;
+    font-size: 22px;
+    font-weight: 700;
+    letter-spacing: 2px;
+    line-height: 22px;
+    color: $menu-text-color;
+    font-family: "Inter", sans-serif;
+    text-transform: uppercase;
```

Comment:
You used some of this styles in &__phona and &__test-ride, maybe it will be better if you create placeholder or mixin